tutorials/cp-cf-workflowmanagement-discover/cp-cf-workflowmanagement-discover.md

```diff
@@ -41,6 +41,7 @@ In this tutorial, you can import a sample package with different artifacts using
 
     !![Edit Destination](cp-cf-wm-discover-editdestination.png)
     !![Destination Password](cp-cf-wm-discover-destinationpassword.png)
+    > Ensure that the credentials of the trial user that are entered in the destination does not have **Two Factor Authentication enabled**.
```

Comment:
I think it should say "do not have" when referring to credentials.

Reply:
Thank you for your input. This is now modified. please kindly merge the changes,